Apps/DiceRoller/DiceRoller/ISDRProjectManager.m

```diff
@@ -24,15 +24,77 @@ - (id)initWithNibName:(NSString *)nibNameOrNil bundle:(NSBundle *)nibBundleOrNil
     return self;
 }
 
+
 - (void)viewWillAppear:(BOOL)animated {
     
     [super viewWillAppear:animated];
     
     dm = [DataManager getInstance];
-    int curProjID = [dm getProjectID];
-    NSString *curProjIDStr = (curProjID > 0) ? [NSString stringWithFormat:@"%d", curProjID] : kNO_PROJECT;
     
-    [projectLbl setText:[NSString stringWithFormat:@"Uploading to Project: %@", curProjIDStr]];
+    pm = [ [ISDRProjectManager alloc] init];
+    
+    bool projectValid = [pm projectHasValidFields];
+    
+    if( projectValid ){
```

Comment:
Me being picky:
Line 38 ISDRProjectManager.m should be:
```if (projectValid) {```
- Space goes after `if`, no space between parens and inner contents, space after closing paren